R/data.R

```diff
@@ -4,8 +4,8 @@
 #'
 #' @format The \code{MI} object is a set of data sets which was used to estimate the causal effect of body mass index and type 2 diabetes on the risk of myocardial infarction.
 #' \describe{
-#'    \item{MI.marginal.Amatrix}{The marginal Â matrix. Column one and two are the marginal estimates of the SNPs on body mass index from GIANT consortium (n = 339,224) (Locke et al., 2015) and type 2 diabetes from DIAGRAM+GERA+UKB (n = 659,316) (Xue et al., 2018), respectively}
-#'    \item{MI.Amatrix}{The conditional Â matrix composed by JAM and the marginal Â matrix. Column one and two are the conditional effect estimates of the SNPs on body mass index and type 2 diabetes, respectively.}
```

Comment:
@lailylajiang LaTeX symbols and text should be included using either \eqn{}, which is equivalent to $$, or \deqn{}, which is equivalent to $$$$. eqn and deqn have two arguments, the second one optional. The first argument is the latex code you want to include, and the second argument is the ascii equivalent, which is usually a verbatim way to represent the symbol. The latex text will be included in the pdf manual of the package, whereas the ascii is included in the help files.

Reply:
Got it